taichi/transforms/auto_diff.cpp

```diff
@@ -989,6 +989,9 @@ class MakeAdjoint : public ADTransform {
 
   void visit(GlobalLoadStmt *stmt) override {
     // issue global store to adjoint
+    if (stmt->src->is<ExternalPtrStmt>()) {
+      TI_ERROR("Src of GlobalLoadStmt does not support ExternalPtrStmt now.")
```

Comment:
How about "Exporting/importing data to/from external array (such as numpy array) not supported in AutoDiff for now." ?